packages/hub/src/lib/dataset-info.ts

```diff
@@ -27,7 +31,7 @@ export async function datasetInfo<
 	]).toString();
 
 	const response = await (params.fetch || fetch)(
-		`${params?.hubUrl || HUB_URL}/api/datasets/${params.name}?${search.toString()}`,
+		`${params?.hubUrl || HUB_URL}/api/datasets/${params.name}${params.revision ? `/revision/${params.revision}` : ''}?${search.toString()}`,
```

Comment:
Alternative:

```suggestion
		`${params?.hubUrl || HUB_URL}/api/datasets/${params.name}/revision/${encodeURLComponent(params.revision ?? "HEAD")}?${search.toString()}`,
```

Reply:
I applied the change! Thanks for the solution